dbt/include/databricks/macros/relations/liquid_clustering.sql

```diff
@@ -0,0 +1,37 @@
+{% macro liquid_clustered_cols() -%}
+  {%- set cols = config.get('liquid_clustered_by', validator=validation.any[list, basestring]) -%}
+  {%- if cols is not none %}
+    {%- if cols is string -%}
+      {%- set cols = [cols] -%}
+    {%- endif -%}
+    CLUSTER BY (
+    {%- for item in cols -%}
```

Comment:
can you use cols | join(', ')

Reply:
this is the value of bringing in a fresh set of eyes...I've just been copy pasting this pattern, but I suspect your suggestion will work.  Trying now.

Reply:
Yep, totally works.  Will do another PR where I replace this pattern elsewhere, as join is much cleaner.